docs/source/data/resolvers.mdx

```diff
@@ -77,12 +77,8 @@ const users = [
 ];
 ```
 
-<!-- TODO (AS4): add this back in once data sources are up
-
 > To learn how to fetch data from an external source (like a database or REST API), see [Data sources](./data-sources/).
```

Comment:
Maybe leave in or update the TODO here? Or will we have a data-sources doc?

Reply:
I'll leave the TODO in and add a link to "Fetching from REST" for now.